content/200-concepts/100-components/01-prisma-schema/08-views.mdx

```diff
@@ -273,15 +272,14 @@ In order to make this `view` block valid you will need to:
 - Remove the comment Prisma generated warning about an invalid view
 
 ```prisma highlight=4;add|1,3,8,9;delete
-/// The underlying view does not contain a valid unique identifier and can therefore currently not be handled by the Prisma Client.
+/// The underlying view does not contain a valid unique identifier and can therefore currently not be handled by Prisma Client.
 view UserInfo {
-  id    Int?
-  id    Int    @unique
-  email String?
-  name  String?
-  bio   String?
-  
-  @@ignore
+id    Int?
+id    Int    @unique
+email String?
+name  String?
+bio   String?
+@@ignore
```

Comment:
broken indentation

Reply:
Resolved in this PR: https://github.com/prisma/docs/pull/4690